### DISCUSSION TITLE
Add UI Support for Multiple Workspaces

### DISCUSSION THREAD

**Discussion by gokulchittaranjan** [airbytehq/airbyte#36065] [Connector Ideas and Features]
## Tell us about the problem you're trying to solve
The airbyte-webapp currently supports only the default workspace.
The workspace-id is hardcoded.

```airbyte
    workspaceId: string;
    workspaceId: "5ae6b09b-fdec-41af-aaf7-7d94cfc33ef6"
```

This makes all the sources/destinations/connections from other workspaces invisible. Currently there is no feature to change the workspace. 

## Describe the solution you’d like
Ideally, it should be possible to list and manage workspaces.

Less than ideal is to have some quick and dirty way to atleast change the workspace ID used by the UI - either through the settings page, or via a command/setting a variable in the developer's console of the browser.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368165818627) by [Unito](https://www.unito.io)

**Comment by jrhizor** [comment]
It seems like all powered-by-airbyte users are asking for this.

**Comment by sivankumar86** [comment]
We are interested in this feature. I donot want to maintain a app which contains workspace  url to redirect to airbyte as it is working from rest but, need to have option from GUI.

[a comment by an automated account is not shown]
